comps/comps-katello-pulp-server-rhel6.xml

```diff
@@ -23,10 +23,6 @@
        <packagereq type="default">pulp-consumer-client</packagereq>
        <packagereq type="default">pulp-docker-plugins</packagereq>
        <packagereq type="default">pulp-docker-admin-extensions</packagereq>
-       <packagereq type="default">pulp-nodes-admin-extensions</packagereq>
-       <packagereq type="default">pulp-nodes-child</packagereq>
```

Comment:
I think this may break capsule installs due to https://github.com/Katello/puppet-pulp/blob/master/manifests/child/install.pp#L5

i'm not 100% of the best way to do it because if katello doesn't use pulp::child class there lots of other things aren't configured properly